server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -263,6 +278,224 @@ TransportResponse handleExtensionRequest(ExtensionRequest extensionRequest) {
         return null;
     }
 
+    public void getNamedWriteables() {
+
+        // retrieve named writeable registry entries from each extension
+        for (DiscoveryNode extensionNode : extensionsList) {
+            try {
+                Map<DiscoveryNode, Map<Class<?>, Map<String, ExtensionReader>>> extensionRegistry = getNamedWriteables(extensionNode);
+                if (extensionRegistry.isEmpty() == false) {
+                    this.extensionNamedWriteableRegistry.putAll(extensionRegistry);
+                }
+            } catch (UnknownHostException e) {
+                logger.error(e.toString());
+            }
+        }
+
+        // TODO : enable dynamic registration of named writeables by returning the extension registry map within Node.java
```

Comment:
If this is not the part of the PR. Let's create an issue and link here to not miss the work

Reply:
Great comment/advice, @owaiskazi19.  Any temporary placeholder or TODO code should have an issue tracking it and put the link to the issue in a comment.